modules/ting_das/ting_das.info

```diff
@@ -0,0 +1,8 @@
+name = Ting digital article service
+description = Use DAS from KB to order articles.
```

Comment:
This seems a bit internal (KB, DAS etc.). Consider dropping acronyms and elaborating.

Reply:
Fixed.